GraphicEditor/MapObject.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+namespace GraphicEditor
+{
+    class Rectt
+    {
+        public String Id { get; set; }
+        public String RoomType { get; set; }
+        public int Width { get; set; }
+        public int Height { get; set; }
+        public int Left { get; set; }
+        public int Top { get; set; }
+        public SolidColorBrush Color { get; set; }
+    }
+    class MapObject
+    {
+
+        public MapObject()
+        {
+
+        }
+
+        public SolidColorBrush Border { get; private set; }
+
+        public List<Rectt> LoadFromFile(string textFile)
+        {
+            List<Rectt> rectangles = new List<Rectt>();
+            if (File.Exists(textFile))
+            {
+                // Read a text file line by line.  
+                string[] lines = File.ReadAllLines(textFile);
+                foreach (string line in lines)
+                {
+                    string[] parts = line.Split(',');
+                  //  byte b1 = Byte.Parse(parts[6]);
+                   // byte b2 = Byte.Parse(parts[7]);
+                    //byte b3 = Byte.Parse(parts[8]);
+
+                    //Color col = Color.FromRgb(b1, b2, b3);
+
+                    rectangles.Add(new Rectt()
+                    {
+                        Id = parts[0],
+                        RoomType = parts[1],
+                        Width = Int32.Parse(parts[2]),
+                        Height = Int32.Parse(parts[3]),
+                        Left = Int32.Parse(parts[4]),
+                        Top = Int32.Parse(parts[5])
+
+                    });
+                }
+            }
+            return rectangles;
+        }
+        public List<Rectangle> Window_Loaded(string textFile, Canvas canvas)
+        {
+            // ... Create list of our Rect objects.
+            List<Rectt> rects = new List<Rectt>();
+            rects = LoadFromFile(textFile);
+
+            List<Rectangle> rectangles = new List<Rectangle>();
+            foreach (Rectt rect in rects)
+            {
+                // ... Create Rectangle object.
+                Rectangle r = new Rectangle();
+                r.Uid = rect.Id;
+                r.Name = rect.RoomType;
+                r.Width = rect.Width;
+                r.Height = rect.Height;
+                Color Black = Color.FromRgb(0, 0, 0);
```

Comment:
if variable is used in more than 4 lines, use more meaningful name than just 'r'.